action/lib/parse.js

```diff
@@ -24,18 +50,77 @@ export default function (title, target) {
     return process.exit(0) // soft exit
   }
 
+  let isDev = devDependencyRegEx.test(title);
+  let isProd = dependencyRegEx.test(title);
+  const isSecurity = securityRegEx.test(title) || labels.includes("security") || labels.includes("Security");
+
+  if (!isDev && !isProd) {
+    // couldn't extract the dependency type from the title, try to read package.json
+    try {
+      const packageJsonPath = path.join(ghWorkspace, 'package.json')
+      const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, "utf8"));
+      isDev = !!packageJson.devDependencies && depName in packageJson.devDependencies
```

Comment:
`!!` is not really needed here ... can simply be `packageJson.devDependencies && ...`

Reply:
That is the TypeScript side in me coming out. Without `!!`, `isDev` is `undefined | boolean`. With it, it is just `boolean`. I can leave it out.